src/apps/settings/src/app/components/Nav/SettingsNav.tsx

```diff
@@ -60,12 +60,30 @@ export const SettingsNav = () => {
         categories.add(setting.category)
       );
 
-      return Array.from(categories)?.map((category) => ({
-        label: startCase(category.replace(/_|-/g, " ")),
-        path: `/settings/instance/${category}`,
-        icon: SettingsRoundedIcon,
-        children: [],
-      }));
+      const instanceSettingsCategories = Array.from(categories)?.map(
+        (category) => ({
+          label: startCase(category.replace(/_|-/g, " ")),
+          path: `/settings/instance/${category}`,
+          icon: SettingsRoundedIcon,
+          children: [],
+        })
+      );
+
+      // Makes sure that the Bynder settings item is present if the user hasn't added any Bynder integration setting yet
```

Comment:
So this is to make sure that the Bynder nav item is present regardless if the setting has been set up yet or not. Since the bynder category won't be part of the response when they hit `/env/settings` on their instance initially.